src/main/java/Group15/ViewController.java

```diff
@@ -10,6 +10,7 @@ public class ViewController
     public static void init()
     {
         ViewController.stage = new Stage();
+        ViewController.stage.setScene(ExerciseDetailsView.createScene());
```

Comment:
We should probably not set the scene to the detailView when the viewController is initialized